config/schema.go

```diff
@@ -63,7 +63,7 @@ func getConfigSchema() *schema.TypedScopeSchema[*Config] {
 					nil,
 					nil,
 					nil,
-					schema.PointerTo(`{"image": {"deployer_name": "docker"}}`),
+					schema.PointerTo(`{"image": {"deployer_name": "podman"}}`),
```

Comment:
Can we specify **both** Podman _and_ Docker?  Would that allow the workflow to select either one?  If the workflow doesn't make a choice, would it just get the first one?

What if we specify only Podman here and Podman is not installed on the platform?  That won't prevent a user from using a config file to use Docker, instead, right?  So, there is no problem with specifying values here which are unusable, right?  So...why not specify both (provided that the default (first?) choice is Podman)?

Can a **workflow** file specify a deployer which is not spec'd by either the config file or the engine defaults?

Reply:
Not as designed. We need to add this at some point.

Reply:
To answer my own question, the configuration can include more than one deployer, but they have to be different types, and `podman` and `docker` are the same type, `image`, so we cannot have more than one of them.